app/src/main/java/com/github/sdp/ratemyepfl/model/items/Course.kt

```diff
@@ -21,14 +21,20 @@ data class Course(
     }
 
     companion object {
+        /**
+         * Converts the json data in a Course object
+         *
+         * @return a course object if it contains the correct data, null otherwise
+         */
         fun DocumentSnapshot.toCourse(): Course? {
-            return try {
```

Comment:
I thought the previous way was pretty cool :(

Reply:
Smart ;)